plugins/content/caption/search/lib/kCaptionSearchFlowManager.php

```diff
@@ -11,8 +11,8 @@ class kCaptionSearchFlowManager implements kObjectDataChangedEventConsumer, kObj
 	public function shouldConsumeAddedEvent(BaseObject $object)
 	{
 		if(class_exists('CaptionAsset') && $object instanceof CaptionAsset 
-				&& CaptionSearchPlugin::isAllowedPartner($object->getPartnerId()
-					&& $object->getStatus() == CaptionAsset::ASSET_STATUS_READY)){
+				&& CaptionSearchPlugin::isAllowedPartner($object->getPartnerId())
+					&& $object->getStatus() == CaptionAsset::ASSET_STATUS_READY && $object->getLanguage() != CaptionPlugin::MULTI_LANGUAGE){
```

Comment:
argument in isAllowedPartner function is currently invalid , needs to be considered in regression tests.